labellab-flask/api/controllers/issuecontroller.py

```diff
@@ -340,13 +375,49 @@ class FetchTeamIssuesView(MethodView):
      URL:- /api/v1/issue/<int:project_id>/team/<int:team_id>
     """
     @jwt_required
+    @project_member_only
     def get(self,project_id, team_id):
         try:
-                        
+            issue_team = fetch_all_issue_by_team_id(project_id,team_id)  
```

Comment:
Add the check to see if project_id and team_id are there. As you already did the check at one place

Reply:
How can i add a check for team_id?

Reply:
Just check if team_id is provided or not. Similar to how you're checking project_id at line 122